src/Internal.AspNetCore.Sdk/build/AssemblyInfo.Serviceable.cs

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+[assembly: System.Reflection.AssemblyMetadata("Serviceable", "True")]
```

Comment:
Right. This is for regular projects. we started deleting AssemblyInfo.cs from our projects because all of its content is generated by Microsoft.NET.SDK (with the exception of the this serviceable attrbute).